File: python-p/7.24.py
##7-24 约分最简分式 
'''
2021.10.12
19.24.24
By Xcl
'''
a,b=map(int, input().split('/'))
i=b
if(a!=b):
    while i>2:
        i=i-1
        if((a%i==0) and (b%i==0)):
            a=a/i
            b=b/i
else:
    a=1
    b=1
a=int(a)
b=int(b)
print(f'{a}/{b}',end="")

File: java-i/7.24.java
//7-24 约分最简分式 
/*
2021.10.12
18.57.35
By Xcl
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
		String[] str = s.split("/");//分割字符串为两个整数
		int a = Integer.parseInt(str[0]);
		int b = Integer.parseInt(str[1]);
        for(int i=b;i>2;i--){
            if((a%i==0)&&(b%i==0)){
                a=a/i;
                b=b/i;
            }
        }
        System.out.print(a+"/"+b);
    }
}